### PR TITLE
Update requirements section

### PR DESCRIPTION
**Pull request by teddyding**

### CODE REVIEW

File: pages/validators/requirements.md [hunk 1]
```diff
@@ -17,4 +17,34 @@ For your node to successfully ingest bridge transactions from the relevant block
 💡IMPORTANT💡:The RPC endpoint you choose *MUST* satisfy the following requirements
 * supports `eth_chainId` method
 * supports `eth_getLogs` method
-* supports `"finalized"` as an input to `toBlock` parameter (Erigon is currently the only major Ethereum node software that does not support this.)
\ No newline at end of file
+* supports `"finalized"` as an input to `toBlock` parameter (Erigon is currently the only major Ethereum node software that does not support this.)
+
+# Required Configs
+
+💡IMPORTANT💡 The dYdX Chain has important node configurations required for normal chain operation. This includes:
```
Comment (BrendanChou) [inline pages/validators/requirements.md:24]: This is already in a page called `Required`; I don't think we need to call out individual line items as important

File: pages/validators/requirements.md [hunk 2]
```diff
@@ -17,4 +17,34 @@ For your node to successfully ingest bridge transactions from the relevant block
...
+[consensus]
+timeout_commit = "500ms"
+```
+
+## `app.toml`
+
+```
+# Set up correct minimum-gas-price correctly. 
+# `12500000000adv4tnt` is for dYdX testnets. Recommended values may be different for production networks.
+minimum-gas-prices = "0.025ibc/8E27BA2D5493AF5636760E354E46004562C46AB7EC0CC4C1CA14E9E20E2545B5,12500000000adv4tnt"
```
Comment (teddyding) [inline pages/validators/requirements.md:38]: Using `12500000000` to be more consistent with recommendations [here](https://docs.dydx.trade/mainnet/genesis/starting_the_network#minimum-gas-price)

Comment (BrendanChou) [inline pages/validators/requirements.md:38]: these are network-specific right?

**Review by ttl33 — APPROVED** [review]
approving this to unblock for now

File: pages/validators/requirements.md (hunk 2 above)
Comment (teddyding) [reply pages/validators/requirements.md:38]: Yes, included the most info and also comments that `Recommended values may be different for production networks`. IMO this is the best we could do

**Pull request merged**